scripts/e2e/cmd/runner/lfu_one_namespace_one_ingress_test.go

```diff
@@ -52,6 +55,47 @@ var _ = Describe("LFU", func() {
 			err = clientset.CoreV1().Namespaces().Delete("test-brownfield-ns", deleteOptions)
 			Expect(err).To(BeNil())
 		})
-	})
 
+		It("[sub-resource-prefix] should be use the sub-resource-prefix to prefix sub-resources", func() {
+			env := GetEnv()
+			klog.Infof("'subResourceNamePrefix': %s", env.SubResourceNamePrefix)
+			Expect(env.SubResourceNamePrefix).ToNot(Equal(""), "Please make sure that environment variable 'subResourceNamePrefix' is set")
+
+			namespaceName := "e2e-sub-resource-prefix"
+			ns := &v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: namespaceName,
+				},
+			}
+			klog.Info("Creating namespace: ", namespaceName)
+			_, err = clientset.CoreV1().Namespaces().Create(ns)
+			Expect(err).To(BeNil())
+
+			SSLE2ERedirectYamlPath := "testdata/one-namespace-one-ingress/ssl-e2e-redirect/app.yaml"
+			klog.Info("Applying yaml: ", SSLE2ERedirectYamlPath)
+			err = applyYaml(clientset, namespaceName, SSLE2ERedirectYamlPath)
+			Expect(err).To(BeNil())
+			time.Sleep(30 * time.Second)
+
+			gateway, err := getGateway()
+			Expect(err).To(BeNil())
+
+			prefixUsed := false
```

Comment:
if all the listeners are supposed to be prefixed, then it makes more sense to check if any listener doesn't have the prefix